benchopt/utils/terminal_output.py

```diff
@@ -49,7 +49,7 @@ def print_normalize(msg, endline=True, verbose=True):
         MIN_LINE_LENGTH, shutil.get_terminal_size((100, 24)).columns
     )
     n_colors = msg.count('\033') // 2
-    msg = msg.ljust(line_length + n_colors * 11)
+    msg = msg.ljust(line_length + n_colors * 11 - 1)
```

Comment:
otherwise every msg is on its own line @tomMoral

Reply:
I have not understood why this fixed the issue, and it still does not work for @Badr-MOUFAD so I will revert this one